.github/workflows/ubuntu-20.04.yml.extra

```diff
@@ -0,0 +1,284 @@
+# vim: set ft=yaml:
```

Comment:
This file has the few portions of the `main.yml` pipelines that ran on Linux but only for one version.
It gets appended to only one of the templated out files (ubuntu-20.04.yml)